spec/services/client_login_service_spec.rb

```diff
@@ -83,6 +83,27 @@
           expect(subject.intakes_for_token("hashed_token")).to match_array [matching_intake]
         end
       end
+
+
+      context "with US intakes" do
+        subject { described_class.new(:statefile) }
+
+        it "returns the az intake with matching phone number" do
+          create :text_message_access_token, token: "hashed_token", sms_phone_number: "+16505551212"
+          matching_intake = create :state_file_ny_intake_after_transfer, phone_number: "+16505551212"
```

Comment:
Did you mean to create a NY one here? or does it not affect the result?

Reply:
Good catch